### DISCUSSION TITLE
Crud Service: TTL Index in Fast Data Projection deployed as 'normal'

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#107] [Q&A] [answered]
Hi

If I create an index of type `TTL` in a projection defined in the console Fast Data section, then the resulting `.json` located at `config-maps/crud-service/<projection-name>.json` has that index listed as `normal`.

**Expected Result:**
![image](https://user-images.githubusercontent.com/38134891/203099489-09f04489-f9c7-413f-96c4-40315ffed411.png)
**Actual Result:**
```
{
      "name": "updatedAt",
      "type": "normal",
      "unique": false,
      "fields": [
        {
          "name": "updatedAt",
          "order": 1
        }
      ]
}
``` 
This seems to be an error when the console configuration is saved and the config-maps are generated by the Console Backend.

**Comment by frengooo** [comment, marked as answer]
Hi @giovf96, I think you should create a bug on the [Fast Data board](https://makeitapp.atlassian.net/jira/software/c/projects/FDPB/boards/774) for this issue.

**Reply by danibix95** [reply]
Hi, thank you for reporting this bug. I can confirm it has been fixed and now TTL indexes on projections collections get created as expected.

**Comment by fedepini** [comment]
Hi @giovf96,

thanks for the catch, we are working on this bug.

In particular, we are making sure that the `expireAfterSeconds` option is always included when a `TTL` index is created for Projections. It is possible that the type will still be converted to `normal` by the console backend, as it is already done for all type of indexes in Single Views, Projections and CRUD service.
